### ISSUE TITLE
cloudflared config directory moved from /usr/local/bin/cloudflared to /usr/bin/cloudflared

### ISSUE THREAD

**Issue by sokolbpg** [cloudflare/cloudflared#862]
**Describe the bug**
In the latest update of the rpm package for RHEL version 2022.12.1 (https://github.com/cloudflare/cloudflared/releases/latest/download/cloudflared-linux-x86_64.rpm) the path for the cloudflared start config was moved from /usr/local/bin/cloudflared to /usr/bin/cloudflared.  For those who use the cloudflared.service the "ExecStart" argument was previously set to /usr/local/bin/cloudflared and after updating to the latest version the service will fail to start with the following error:
```

INF Cannot determine default configuration path. No file [config.yml config.yaml] in [~/.cloudflared ~/.cloudflare-warp ~/cloudflare-warp /etc/cloudflared /usr/local/etc/cloudflared]
INF Version 2022.12.1
INF GOOS: linux, GOVersion: go1.19.3, GoArch: amd64
INF cloudflared will not automatically update when run from the shell. To enable auto-updates, run cloudflared as a service: https://developers.cloudflare.com/cloudflare-one/connections/connect-apps/run-tunnel/as-a-service/
INF Cannot determine default origin certificate path. No file cert.pem in [~/.cloudflared ~/.cloudflare-warp ~/cloudflare-warp /etc/cloudflared /usr/local/etc/cloudflared] originCertPath=
ERR You need to specify the origin certificate path with --origincert option, or set TUNNEL_ORIGIN_CERT environment variable. See https://developers.cloudflare.com/argo-tunnel/reference/arguments/ for more information. originCertPath=
ERR Couldn't start tunnel error="Error getting origin cert: client didn't specify origincert path when running from terminal"
```

**To Reproduce**
1. Update cloudflared (ex: curl -L https://github.com/cloudflare/cloudflared/releases/latest/download/cloudflared-linux-x86_64.rpm -o cf.rpm)
2. sudo yum install ./cf.rpm
3. sudo systemctl start cloudflared 

**Expected behavior**
I would expect the location of the config file to not change out of the blue.  This change is not documented and results in a total DNS failure.

**Environment and versions**
 - OS: AlmaLinux 9.1
 - Architecture: Intel 64 bit
 - Version: 2022.12.1

**Logs and errors**
```

INF Cannot determine default configuration path. No file [config.yml config.yaml] in [~/.cloudflared ~/.cloudflare-warp ~/cloudflare-warp /etc/cloudflared /usr/local/etc/cloudflared]
INF Version 2022.12.1
INF GOOS: linux, GOVersion: go1.19.3, GoArch: amd64
INF cloudflared will not automatically update when run from the shell. To enable auto-updates, run cloudflared as a service: https://developers.cloudflare.com/cloudflare-one/connections/connect-apps/run-tunnel/as-a-service/
INF Cannot determine default origin certificate path. No file cert.pem in [~/.cloudflared ~/.cloudflare-warp ~/cloudflare-warp /etc/cloudflared /usr/local/etc/cloudflared] originCertPath=
ERR You need to specify the origin certificate path with --origincert option, or set TUNNEL_ORIGIN_CERT environment variable. See https://developers.cloudflare.com/argo-tunnel/reference/arguments/ for more information. originCertPath=
ERR Couldn't start tunnel error="Error getting origin cert: client didn't specify origincert path when running from terminal"
```

**Comment by sokolbpg** [comment]
@joliveirinha 

My apologizes on my terminology.  I am referring to the binary location, not the config location - the error log mentioned something about the config.yaml file not being found, which is connected to the binary being moved.  I have had cloudflared installed for many months without issue and had automatic updates configured (installed cloudflared the manual way).  When cloudflared automatically updated to version 2022.12.1, I found that my pi-hole was no longer able to resolve DNS since the cloudflared service was not starting.  Cloudflared would attempt to start and then error out and fail to start.  Thus, the reason for a total DNS failure.

The reason it failed to start was because it was looking in the /usr/local/bin/cloudflared directory based on the ExecStart command in /etc/systemd/system/cloudflared.service specifying this path: /usr/local/bin/cloudflared based on the documentation located here: https://docs.pi-hole.net/guides/dns/cloudflared/.  However, when checking that path I found it was empty.  I proceeded to do a clean install (removed the existing installation and installed again).  However, as you can see below the /usr/local/bin/cloudflared directory is still empty even though the binary had been stored here for years.

[root@pi-hole ~]# ll /usr/bin/cloudflared
-rwxr-xr-x. 1 cloudflared cloudflared 35232173 Dec 20 07:56 /usr/bin/cloudflared
[root@pi-hole ~]# ll /usr/local/bin/cloudflared
ls: cannot access '/usr/local/bin/cloudflared': No such file or directory
[root@pi-hole ~]#

After changing the ExecStart path to point to /usr/bin/cloudflared the service was able to start successfully.

TLDR; I am trying to start the cloudflared service, it is failing because it cannot find the binary, and changing the ExecStart path to point to the location the binary is now stored resolves the issue.

The binary location shouldn't change out of the blue, which is why I created this bug.

**Comment by sudarshan-reddy** [comment]
Hey @sokolbpg ! 

If you look at the changelog, we haven't changed where the binary gets installed at all. However, an important question in finding out what the problem is : "Did the way you install cloudflared change?".

This would affect where cloudflared got installed. For example, there is a post script if you yum install that does a symlink to the local path. Either: 

1. This symlink did not fire (could be a permission problem).
2. Or it was not even triggered by a different installation method.

**Comment by dpdug4n** [comment]
Ran into the same issue on PopOS after following the debian/ubuntu install instructions from PiHole: https://docs.pi-hole.net/guides/dns/cloudflared/
Copying the binary to the expected path resolved the issue.
`sudo cp /usr/bin/cloudflared /usr/local/bin/cloudflared
`
